__tests__/fetcher/api_mockdata.ts

```diff
@@ -0,0 +1,2851 @@
+export const articleUuidMock = {
+  __typename: 'Article',
```

Comment:
Such a file should be placed in a directory `__fixtures__`. See https://github.com/serlo/api.serlo.org/tree/master/__fixtures__ for an example.

Reply:
put it in `___tests___/___fixtures___` because it only concerns the tests currently.

Reply:
nevermind, is in root now, so we need no config change